tools/syz-linter/linter.go

```diff
@@ -382,3 +384,23 @@ func (pass *Pass) nodeString(n ast.Node) string {
 	printer.Fprint(w, pass.Fset, n)
 	return w.String()
 }
+
+func (pass *Pass) checkAssignStmt(n *ast.AssignStmt) {
+	if len(n.Lhs) != len(n.Rhs) {
+		return
+	}
+	for i, lhs := range n.Lhs {
+		lIdent, ok := lhs.(*ast.Ident)
+		if !ok {
+			return
+		}
+		rIdent, ok := n.Rhs[i].(*ast.Ident)
+		if !ok {
+			return
+		}
+		if lIdent.Name != rIdent.Name {
+			return
+		}
+	}
+	pass.report(n, "Loops are fixed since go122 https://go.dev/blog/loopvar-preview")
```

Comment:
It would be useful to say what exactly is wrong with the code, or how should it be changed.
The current message is not very useful for programmer.